### ISSUE TITLE
Output declarations with a glob pattern fail when AWS Batch executor is used

### ISSUE THREAD

**Issue by pditommaso** [nextflow-io/nextflow#748]
A process defining one or more output files declarations containing a glob pattern, are not correctly resolved when using the AWS Batch executor and the process execution fail. 

For example: 

```nextflow
process foo {
  output: file '*.{zip,html}'
  '''
  command_here
  '''
}
```

Produced the following error 

```
ERROR ~ Error executing process > 'foo'

Caused by:
  Missing output file(s) `*.{zip,html}` expected by process `foo`
```

**Comment by pditommaso** [comment]
This is a regression caused by #171

**Comment by pditommaso** [comment]
Available in `0.30.2`.
